config/README.md

```diff
@@ -1,102 +1,103 @@
-# Multi-region cockroachdb setup
+# Deploying a DSS
+
+## Preface
+
+This doc provides a well-lit path for deploying the DSS and its dependencies (
+namely CockroachDB) on Kubernetes. This is not a requirement, and a DSS node
```

Comment:
Let's pick a consistent terminology for referring to a single DSS instance/node.  I think "DSS instance" is generally more common that "DSS node", "instance" helps differentiate between CRDB nodes and DSS instances, and "DSS instance" is used above, so I would recommend "DSS instance" over "DSS node".  If we really meant to say "CRDB node" here rather than DSS node (since a DSS instance only joins a cluster via its CRDB nodes), let's clarify that.

Reply:
from past teams i think we should avoid the word "instance" at all costs, as it's probably one of the most ambiguous and over used words in software engineering :)

Added a glossary to explain the terminology i'm using in this doc.